zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -62,15 +62,17 @@
         <li ng-if="ticket">
         <!--TODO(bzz): move to Typeahead https://angular-ui.github.io/bootstrap  -->
           <form role="search"
-            style="width: 300px; display: inline-block; margin: 0 10px"
+            style="display: inline-block; margin: 0 10px"
+             class="navbar-form"
              ng-submit="search(searchTerm)">
             <div class="input-group">
               <input
                 type="text"
+                style="min-width:300px;"
                 ng-model="searchTerm"
                 ng-disabled="!navbar.connected"
                 class="form-control"
-                placeholder="Search in your notebooks"
+                placeholder="Search for your Notebooks"
```

Comment:
I think, this is actually "in" - we searching for text matches in text, name, title and so on.
Perhaps say "Search your notebooks"?